plugins/inputs/mongodb/mongodb_data.go

```diff
@@ -246,6 +246,9 @@ func (d *MongodbData) AddDefaultStats() {
 			d.add(key, floatVal)
 		}
 		d.addStat(statLine, WiredTigerExtStats)
+		if d.StatLine.FaultsCnt > 0 {
```

Comment:
Would it be wrong to report 0 here?

Reply:
I don't think there's any _harm_ in sending 0.  I was trying with this stat to follow the existing patterns where the values were sent only if they were non-null, or non-empty; I'm not sure how valuable it is in this case.  I'm not even sure, in this instance, if the serverStatus output always includes the value or not.  If you prefer, I can just send it every time.

Reply:
Yeah, let's change it to always send.